fastdtw/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .fastdtw import fastdtw
+from .fastdtw import dtw
+from .fastdtw_py import fastdtw as fastdtw_py
```

Comment:
Can you make sure in the _test.py_ file that the two implementations give the same (correct) result?